Add access rule examples for remote exams (#2224)
PrairieLearn/PrairieLearn: doc/remoteExams.md

# Remote exam configurations

This page lists sample assessment configurations for remote exams, where students are not physically present in the same location as the proctors.

## Synchronous, timed exams

This configuration is good when:

* Almost all students take the exam at the same time

> * Some students have accommodations, such as 1.5x time -- **these students must be listed first in the access rules**

Reviewer: ```suggestion
* Some students have accommodations, such as 1.5x time &mdash; **these students must be listed first in the access rules**
```

you should use the emdash &mdash; it's extra fancy :slightly_smiling_face:

Author: true, and I even know how to type them — I think I got lazy from the MacOS slack client which auto-converts double-dashes to em-dashes 😄

* Some students take the exam at a later "conflict" time, mainly because they are in a different timezone

Some notes about this configuration:

* The exam window (70 minutes, `startDate` to `endDate`) has been set to be 10 minutes longer than the exam time limit (60 minutes). If a student starts the exam late, then the countdown timer on their exam will show the full exam time limit (60 minutes). However, they will not be able to access the exam past the `endDate` time under any circumstances. See [PL issue #2217](https://github.com/PrairieLearn/PrairieLearn/issues/2217) for details.
* Remember to extend both `endDate` **and** `timeLimitMin` for students with extra-time accommodations.
* Students who are scheduled for a conflict exam will be able to access the exam during the primary time slot. However, if they do so, they will be blocked from the exam during the conflict timeslot.
* After the timer expires the exam will auto-close and grade any saved but ungraded questions and show students their final score. After this time students will be unable to see any of the questions.

> * If a student closes their webbrowser before the exam is complete, their exam will not automatically close. This is not a security problem because it will automatically close if they try and access it again. All exams will auto-close 6 hours after the end of the exam window.

Reviewer: ```suggestion
* If a student closes their web browser before the exam is complete, their exam will not automatically close. This is not a security problem because it will automatically close if they try to access it again. All exams will auto-close 6 hours after the end of the exam window.
```
